projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -24,40 +24,55 @@
+	 * Supported integrations configuration.
+	 *
+	 * Each integration array supports the following keys:
+	 * - type (string)                  : 'plugin' or 'service'
+	 * - file (string|null)             : Plugin file path (for plugins), or null for services
+	 * - settings_url (string|null)     : Relative admin URL for settings, or null if none
+	 * - marketing_redirect_slug (string|null) : Slug for Redirect::get_url() for marketing links, or null if none
+	 *
+	 * For marketing_redirect_slug, you'll need to add those here first:
+	 * https://mc.a8c.com/jetpack-crew/redirects/
 	 *
 	 * @var array
 	 */
 	private $supported_integrations = array(
 		'akismet'                           => array(
-			'type'         => 'plugin',
-			'file'         => 'akismet/akismet.php',
-			'settings_url' => 'admin.php?page=akismet-key-config',
+			'type'                    => 'plugin',
+			'file'                    => 'akismet/akismet.php',
+			'settings_url'            => 'admin.php?page=akismet-key-config',
+			'marketing_redirect_slug' => 'org-spam',
 		),
 		'creative-mail-by-constant-contact' => array(
-			'type'         => 'plugin',
-			'file'         => 'creative-mail-by-constant-contact/creative-mail-plugin.php',
-			'settings_url' => 'admin.php?page=creativemail',
+			'type'                    => 'plugin',
+			'file'                    => 'creative-mail-by-constant-contact/creative-mail-plugin.php',
+			'settings_url'            => 'admin.php?page=creativemail',
+			'marketing_redirect_slug' => 'creative-mail',
 		),
 		'zero-bs-crm'                       => array(
-			'type'         => 'plugin',
-			'file'         => 'zero-bs-crm/ZeroBSCRM.php',
-			'settings_url' => 'admin.php?page=zerobscrm-plugin-settings',
+			'type'                    => 'plugin',
+			'file'                    => 'zero-bs-crm/ZeroBSCRM.php',
+			'settings_url'            => 'admin.php?page=zerobscrm-plugin-settings',
+			'marketing_redirect_slug' => 'org-crm',
 		),
 		'salesforce'                        => array(
-			'type'         => 'service',
-			'file'         => null,
-			'settings_url' => null,
+			'type'                    => 'service',
+			'file'                    => null,
+			'settings_url'            => null,
+			'marketing_redirect_slug' => null,
 		),
 		'google-drive'                      => array(
-			'type'         => 'service',
-			'file'         => null,
-			'settings_url' => null,
+			'type'                    => 'service',
+			'file'                    => null,
+			'settings_url'            => null,
+			'marketing_redirect_slug' => null,
 		),
 		'mailpoet'                          => array(
-			'type'         => 'plugin',
-			'file'         => 'mailpoet/mailpoet.php',
-			'settings_url' => 'admin.php?page=mailpoet-homepage',
+			'type'                    => 'plugin',
+			'file'                    => 'mailpoet/mailpoet.php',
+			'settings_url'            => 'admin.php?page=mailpoet-homepage',
+			'marketing_redirect_slug' => 'org-mailpoet',
```

Comment:
Marketing/learn more links are basically always external links. And for those, we are supposed to use Jetpack redirects per guidelines and fieldguide. By adding a redirect slug rather than url here, we basically enforce that. Before we were being inconsistent.

We are also defining these in the endpoint rather than defining them whenever needed on the frontend to ensure consistency.